src/app/utils.py

```diff
@@ -171,26 +171,44 @@ def makePullRequest(username, token, branchName, updateUpstream, fileName, commi
     """ Creating Commit """
     if fileName[-4:] == ".xml":
         fileName = fileName[:-4]
+    textFileName = fileName + ".txt"
     fileName += ".xml"
     commit_url = "{0}repos/{1}/{2}/contents/src/{3}".format(url, username, settings.NAMESPACE_REPO_NAME if is_ns else settings.LICENSE_REPO_NAME, fileName)
+    text_commit_url = "{0}repos/{1}/{2}/contents/test/simpleTestForGenerator/{3}".format(url, username, settings.NAMESPACE_REPO_NAME if is_ns else settings.LICENSE_REPO_NAME, textFileName)
     xmlText = xmlText.encode('utf-8') if isinstance(xmlText, str) else xmlText
     fileContent = base64.b64encode(xmlText).decode()
+    plainText = plainText.encode('utf-8') if isinstance(plainText, str) else plainText
+    textFileContent = base64.b64encode(plainText).decode()
     body = {
         "path":"src/"+fileName,
         "message":commitMessage,
         "content":fileContent,
         "branch":branchName,
     }
+    text_file_body = {
+        "path":"src/"+ textFileName,
+        "message":commitMessage,
+        "content":textFileContent,
+        "branch":branchName,
+    }
     """ Check if file already exists """
     file_url = "{0}/contents/src/{1}".format(TYPE_TO_URL_NAMESPACE[NORMAL] if is_ns else TYPE_TO_URL_LICENSE[NORMAL], fileName)
     response = requests.get(file_url, headers=headers)
+    text_file_url = "{0}/contents/test/simpleTestForGenerator/{1}".format(TYPE_TO_URL_NAMESPACE[NORMAL] if is_ns else TYPE_TO_URL_LICENSE[NORMAL], textFileName)
+    text_response = requests.get(text_file_url, headers=headers)
     if response.status_code == 200:
         """ Creating Commit by updating the file """
         data = json.loads(response.text)
         file_sha = data["sha"]
         body["sha"] = file_sha
```

Comment:
Let's create a util for this since everything is same for both fileName and textFileName. Call the util along with passing body and return updated body.